scripting/get5/natives.sp

```diff
@@ -126,7 +126,9 @@ public int Native_LoadMatchConfigFromURL(Handle plugin, int numParams) {
   GetNativeString(1, url, sizeof(url));
   ArrayList paramNames = view_as<ArrayList>(GetNativeCell(2));
   ArrayList paramValues = view_as<ArrayList>(GetNativeCell(3));
-  return LoadMatchFromUrl(url, paramNames, paramValues);
+  ArrayList headerNames = view_as<ArrayList>(GetNativeCell(4));
+  ArrayList headerValues = view_as<ArrayList>(GetNativeCell(5));
+  return LoadMatchFromUrl(url, paramNames, paramValues, headerNames, headerValues);
```

Comment:
Since `LoadMatchFromUrl` does not `delete` any of the parameters passed to it, we must:

`delete paramNames;`
`delete paramValues;`
`delete headerNames;`
`delete headerValues;`

after calling `LoadMatchFromUrl`.

Reply:
No, wait, sorry. This is wrong. Whoever calls `Get5_LoadMatchConfigFromURL` should do this.